taskmanager/src/Application/Task/Status.php

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
@partikus for project classes I would suggest strict type

Reply:
you're right, could you provide phpspec templates for classes, please?

Reply:
nice idea created https://github.com/PGS-php/upskilling-2nd-edition/issues/5 and will add or first come, first served

Reply:
@partikus sth like https://github.com/PGS-php/upskilling-2nd-edition/pull/7 ?

Reply:
@dominikhajduk sth like that. :) have you already tested it?

Reply:
@partikus sure merged with your branch locally works great 🥇

Reply:
I'm extremely busy within next 24 hours. Could you contribute to my fork with changes that you requested, please? Otherwise please wait till Monday/Tuesday.

Reply:
@partikus please consider merge in: https://github.com/partikus/upskilling-2nd-edition/pull/2